### PR TITLE
[PR] Install latest npm dependency

### PR DESCRIPTION
**Pull request by schuerle**
yarn audit v1.22.10
0 vulnerabilities found - Packages audited: 280
Done in 1.00s.

Run: `yarn && yarn audit`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,5 +27,8 @@
   "devDependencies": {
     "publish": "0.6.0",
     "winston": ">=2.2.0 <3"
+  },
+  "resolutions": {
+    "publish/npm": "*"
```
Comment (gormed) [inline package.json:32]: This looks weird, doesn't it mean: "use any npm version you like", and thus changes nothing?

How should it behave? I found nothing in the spec for resolutions: https://github.com/yarnpkg/rfcs/blob/master/implemented/0000-selective-versions-resolutions.md :thinking:

**Review by gormed — COMMENTED** [review]
Does this actually fix security scan warnings? If yes i do not get why :sweat_smile:

**Review by gormed — APPROVED** [review]
Satisfies trivy, so :ok_hand: :shrug:

**Comment by schuerle** [comment]
Yeah, sadly the scanner does not provide the dependency-path

**Pull request merged**